src/components/Delegates/DelegateCard/DelegateActions.tsx

```diff
@@ -46,9 +45,8 @@ export function DelegateActions({
   }
 
   return (
-    <HStack
-      alignItems="items-stretch"
-      className={className ? className + "justify-between" : "justify-between"}
+    <div
+      className={`flex flex-row items-stretch ${className ? className + "justify-between" : "justify-between"}`}
```

Comment:
could probably add `justifyBetween` outside of the ternary since it's used by both

Reply:
@mcgingras good catch. Will fix.